pyproject.toml

```diff
@@ -30,6 +30,7 @@ classifiers = [
 dependencies = [
     "beautifulsoup4",
     "biopython",
+    "deprecated",
```

Comment:
Is there maybe a way to work around the dependencies being duplicated in the `requirements.txt` and the `pyproject.toml`?

Reply:
yes, there is [a way](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata). Code udpated.